cogs/displayinfo.py

```diff
@@ -1,19 +1,20 @@
 import discord
 import constantvariables
+import mongoDB
 from discord.ext import commands
 from discord.ext import tasks
 from datetime import datetime
 from datetime import date
+import datetime as dt
 from pymongo.mongo_client import MongoClient
 from pymongo.server_api import ServerApi
 import asyncio
 
 
-# MongoDB URI for connecting to the database
-uri = "mongodb+srv://user:user@rasenbotcluster.toeujwx.mongodb.net/?retryWrites=true&w=majority"
+
 
 # Create a new client and connect to the MongoDB server
-mongoClient = MongoClient(uri, server_api=ServerApi('1'))
+mongoClient = MongoClient(mongoDB.MONGODB_URI, server_api=ServerApi('1'))
```

Comment:
How exactly does this work? Will this always be the URI for RasenBot?

Reply:
Need confirmation from @benjaminl4917 but it looks like a secrets file with the mongo URI was created but not pushed to github.

Reply:
Can someone explain this to me?

Reply:
Would that cause any issues? What if another developer wants to use the bot. Would they need the mongo URI?

Reply:
@benjaminl4917 asking for comment. most likely should have a secrets file with own URI and ask other developers to provide their own URI. we can also have mongoDB run natively (not in the cloud) on the local machine running the bot.

Reply:
> Need confirmation from @benjaminl4917 but it looks like a secrets file with the mongo URI was created but not pushed to github.

yea currently I have it as a secrets file which I am looking into sharing secret files through github. However, I am opening up a new ticket where hopefully we would be hosting the database locally instead of on the cloud.